### PR TITLE
Update to td_agent and plugins to latest stable

### PR DESCRIPTION
**Pull request by willmore**
An encounter with duplicate messaging, similar to [this reported
issue][1] prompted the need to update fluent-plugin-elasticsearch, which
in turn made updating fluentd (td_agent) necessary.

[1]: https://github.com/uken/fluent-plugin-elasticsearch/issues/312

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # docker-fluentd
 Fluentd image with added plugins:
 
-* fluent-plugin-kubernetes_metadata_filter -v 0.27.0
-* fluent-plugin-elasticsearch -v 1.9.5
+* fluent-plugin-kubernetes_metadata_filter -v 0.31.0
+* fluent-plugin-elasticsearch -v 2.6.1
 * fluent-plugin-dogstatsd -v 0.0.6
-* fluent-plugin-forest -v 0.3.3
```
Comment (deiwin) [inline README.md]: Why is this removed from here but not from `build.sh`?

Comment (willmore) [reply README.md]: Accident.
